packages/ui/v2/core/Shell.tsx

```diff
@@ -484,7 +485,9 @@ function MobileNavigationContainer() {
 }
 
 const MobileNavigation = () => {
+  const router = useRouter();
   const isEmbed = useIsEmbed();
+  if (router.route.startsWith("/v2/settings/")) return null;
```

Comment:
We don't want the bottom navigation bar to be displayed when on the settings page

Reply:
we just need to remember to remove "v2" later haha